src/crypto/keyChain.ts

```diff
@@ -54,6 +54,9 @@ export const retrieveKeychain = async (platform: string): Promise<KeyChain> => {
         return keyChainImpl.generic_unix;
       }
     }
+  } else if (platform === 'browser') {
```

Comment:
the polyfill for `os` will return platform as `browser`, something node would never do